cpp/src/arrow/compute/kernels/hash_aggregate_test.cc

```diff
@@ -1013,6 +1013,74 @@ TEST(GroupBy, MinMaxOnly) {
   }
 }
 
+TEST(GroupBy, MinMaxDecimal) {
+  auto in_schema = schema({
+      field("argument0", decimal128(3, 2)),
+      field("argument1", decimal256(3, 2)),
+      field("key", int64()),
+  });
+  // for (bool use_exec_plan : {false, true}) {
+  //   for (bool use_threads : {true, false}) {
```

Comment:
Should this be re-enabled? Or does it not work properly?

Reply:
My bad, I forgot to re-enable this.